gcc/rust/ast/rust-item.h

```diff
@@ -57,7 +57,7 @@ class TypeParam : public GenericParam
 
   // bool has_type_param_bounds;
   // TypeParamBounds type_param_bounds;
-  std::vector<std::unique_ptr<TypeParamBound> >
+  std::vector<std::unique_ptr<TypeParamBound>>
```

Comment:
which version of clang-format are you using?

Reply:
I downloaded a clang-format 10 binary that I am using specially for gccrs. I got it from the clang-format action that the CI uses. There are a lot of differences when running clang-format 11, which is what I have installed on my machine usually. Is that not the right version?